sites/partners/src/applications/PaperApplicationForm/sections/FormPreferences.tsx

```diff
@@ -34,7 +36,7 @@ const FormPreferences = ({ county, preferences, hhMembersOptions }: FormPreferen
     const keys = []
     preferences?.forEach((preference) =>
       preference?.formMetadata?.options.forEach((option) =>
-        keys.push(getPreferenceOptionName(preference?.formMetadata.key, option.key))
+        keys.push(getPreferenceOptionName(option, preference?.formMetadata.key, option.key))
```

Comment:
Just noticing the order of arguments here seem to be reversed from code in the public app. Was that intentional?

Reply:
Great catch! 😅  I checked all other uses of the fxn just to be sure, this one was just flip-flopped.